### PR TITLE
Deployment - Creating new yml file and Updating old yml file

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Review by Pawan-MSFT — APPROVED** [review]
Please fix small suggestion, rest look good, approving

**Review by Pawan-MSFT — APPROVED** [review]
looks correct, approving.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

File: samples/meetings-share-to-stage-signing/csharp/Source/MeetingSigning.Web/Microsoft.Teams.Samples.MeetingSigning.Web.csproj [hunk 1]
```diff
@@ -53,7 +53,6 @@
   <Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish">
     <!-- As part of publishing, ensure the JS resources are freshly built in production mode -->
     <Exec WorkingDirectory="$(SpaRoot)" Command="npm install" />
-    <Exec WorkingDirectory="$(SpaRoot)" Command="npm run build" />
```
Comment (Wajeed-msft) [inline samples/meetings-share-to-stage-signing/csharp/Source/MeetingSigning.Web/Microsoft.Teams.Samples.MeetingSigning.Web.csproj:56]: Why are we removing build from exec command?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline:

File: samples/meetings-share-to-stage-signing/csharp/Source/MeetingSigning.Web/Microsoft.Teams.Samples.MeetingSigning.Web.csproj (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/meetings-share-to-stage-signing/csharp/Source/MeetingSigning.Web/Microsoft.Teams.Samples.MeetingSigning.Web.csproj:56]: I have Reverting this changes.

File: samples/meetings-share-to-stage-signing/csharp/Source/MeetingSigning.Web/Microsoft.Teams.Samples.MeetingSigning.Web.csproj [hunk 2]
```diff
@@ -53,7 +53,7 @@
...
+    <!--<Exec WorkingDirectory="$(SpaRoot)" Command="npm run build" />-->
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/meetings-share-to-stage-signing/csharp/Source/MeetingSigning.Web/Microsoft.Teams.Samples.MeetingSigning.Web.csproj]: I have Reverting this changes.

File: samples/meetings-stage-view/csharp/AppInMeeting/AppInMeeting.csproj [hunk 3]
```diff
@@ -33,7 +33,7 @@
   <Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish">
     <!-- As part of publishing, ensure the JS resources are freshly built in production mode -->
     <Exec WorkingDirectory="$(SpaRoot)" Command="npm install" />
-    <Exec WorkingDirectory="$(SpaRoot)" Command="npm run build" />
+    <!--<Exec WorkingDirectory="$(SpaRoot)" Command="npm run build" />-->
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/meetings-stage-view/csharp/AppInMeeting/AppInMeeting.csproj]: I have I have Reverting this changes and tested deployment pipeline.
image

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline:

**Pull request merged**